pulsar-client-reactive-api/src/test/java/org/apache/pulsar/reactive/client/api/ReactiveMessageConsumerBuilderTests.java

```diff
@@ -55,6 +56,13 @@ class ReactiveMessageConsumerBuilderTests {
 
 	private static final Scheduler scheduler = Schedulers.newSingle("my-sched");
 
+	private static final RedeliveryBackoff redeliverBackoff = new RedeliveryBackoff() {
```

Comment:
I think using a mock here would simplify things a bit but I also see a couple of other fields are creating a "Test<thing>" class and consistency is king. @lhotari what is your opinion on using mock vs. a concrete test implementation class in tests?

Reply:
Cool. The "exponential" has been removed from all test var names, so I am ok w/ this as well.